kafka-plugins-0.10/pom.xml

```diff
@@ -163,6 +167,11 @@
       <artifactId>netty-http</artifactId>
       <version>${netty-http.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.lz4</groupId>
+      <artifactId>lz4-java</artifactId>
```

Comment:
does this work fine with all kafka plugins including streaming source? 
Also does it work in sandbox and cloud environment after making these changes?

Reply:
Have tested on Sandbox : 
 [ BATCH ] : Kafka -> wrangler -> file : works. 
 [ Stream ] : Kafka -> wrangler -> file : works. 
 
 Haven't tested on GCP because of firewall issues.